src/AllContext/AddressContext.js

```diff
@@ -0,0 +1,16 @@
+import { useState, createContext, useContext } from "react";
+import { AddressesDB } from "../Cart/Checkout/AddressManagement/AddressesDB";
+const addressContext = createContext();
+
+export const AddressContextProvider = ({ children }) => {
+  const [addresses, setAddresses] = useState(null);
+  return (
+    <addressContext.Provider value={{ addresses, setAddresses }}>
```

Comment:
Hey, I guess its a convention to use capital letter for react components
```suggestion
    <AddressContext.Provider value={{ addresses, setAddresses }}>
```